### ISSUE TITLE
[Bug] ios safari，when press enter at first line in code fence，will delete the line

### ISSUE THREAD

**Issue by yansaid** [Saul-Mirone/milkdown#200]
ios safari，load the https://milkdown.dev/#/getting-started

then will happend

https://user-images.githubusercontent.com/12773446/143396620-57d70976-f6c5-4379-a22a-75f08d1bd39a.MP4

**Comment by Saul-Mirone** [comment]
Let me have a look

**Comment by Saul-Mirone** [comment]
Seems it's a prosemirror bug, I'll open a issue for them.

**Comment by yansaid** [comment]
thanks for reply，can you link it here?

**Comment by Saul-Mirone** [comment]
I need some help because I still cann't find the root cause, asked in community here:
https://discuss.prosemirror.net/t/bug-ios-safari-when-press-enter-at-first-line-in-code-fence-will-delete-the-line/4249

**Comment by Saul-Mirone** [comment]
Also, I find the same issue on riprap, so I think, it's really a prosemirror bug.

**Comment by philippkuehn** [comment]
Interestingly, this only happens in the vue version of tiptap. the react demo works as expected 🤔

**Comment by Saul-Mirone** [comment]
Fixed by: https://github.com/Saul-Mirone/milkdown/commit/01d9cd3e3b738addc9182a8c8d2e9a8e8cbe926b

I finally find something, it's caused by HTML structure of the code block, if it's:
```html
<pre>
  <code>
  Code Snippets....
  </code>
</pre>
```

Then it will trigger the bug. What I did is insert a `div` after the code tag:
```html
<pre>
  <code>
    <div>
    Code Snippets....
    </div>
  </code>
</pre>
```

**Comment by yansaid** [comment]
> Fixed by: [01d9cd3](https://github.com/Saul-Mirone/milkdown/commit/01d9cd3e3b738addc9182a8c8d2e9a8e8cbe926b)
> 
> I finally find something, it's caused by HTML structure of the code block, if it's:
> 
> ```
> <pre>
>   <code>
>   Code Snippets....
>   </code>
> </pre>
> ```
> 
> Then it will trigger the bug. What I did is insert a `div` after the code tag:
> 
> ```
> <pre>
>   <code>
>     <div>
>     Code Snippets....
>     </div>
>   </code>
> </pre>
> ```

thanks
